exercises/markdown/canonical-data.json

```diff
@@ -120,6 +120,51 @@
             "markdown": "# Start a list\n* Item 1\n* Item 2\nEnd a list"
         },
         "expected": "<h1>Start a list</h1><ul><li>Item 1</li><li>Item 2</li></ul><p>End a list</p>"
+    },
+    {
+        "uuid": "6dca5d10-5c22-4e2a-ac2b-bd6f21e61939",
+        "description": "h7 should not be parsed as it does not exist",
+        "property": "parse",
+        "input": {
+            "markdown": "####### header depth 7 does not exist"
+        },
+        "expected": "####### header depth 7 does not exist"
+    },
+    {
+        "uuid": "9df3f500-0622-4696-81a7-d5babd9b5f49",
+        "description": "with h3 header level",
+        "property": "parse",
+        "input": {
+            "markdown": "### This will be an h3"
+        },
+        "expected": "<h3>This will be an h3</h3>"
+    },
+    {
+        "uuid": "50862777-a5e8-42e9-a3b8-4ba6fcd0ed03",
+        "description": "with h4 header level",
+        "property": "parse",
+        "input": {
+            "markdown": "#### This will be an h4"
+        },
+        "expected": "<h4>This will be an h4</h4>"
+    },
+    {
+        "uuid": "ee1c23ac-4c86-4f2a-8b9c-403548d4ab82",
+        "description": "with h5 header level",
+        "property": "parse",
+        "input": {
+            "markdown": "##### This will be an h5"
+        },
+        "expected": "<h5>This will be an h5</h5>"
+    },
+    {
+        "uuid": "be67e559-74ef-48e9-a4e0-f3aa73208114",
+        "description": "with h6 header level",
+        "property": "parse",
+        "input": {
+            "markdown": "###### This will be an h6"
+        },
+        "expected": "<h6>This will be an h6</h6>"
```

Comment:
This is a duplicate of 13b5f410-33f5-44f0-a6a7-cfd4ab74b5d5

Reply:
Fixed in the newest version. Thanks!